test/RelayHub.test.js

```diff
@@ -0,0 +1,89 @@
+const { BN, ether, expectEvent, time } = require('openzeppelin-test-helpers');
+
+const { getTransactionHash, getTransactionSignature } = require('../src/js/relayclient/utils')
+
+const RelayHub = artifacts.require('RelayHub');
+const SampleRecipient = artifacts.require('SampleRecipient');
+
+contract('RelayHub', function ([_, relayOwner, relay, sender, other]) {
```

Comment:
please check why it fails.
also, please explain why you need to make such a deep context stack, where each one is used only once...

Reply:
I addressed the linter failing in #137, which this PR is built on (it builds on the tests added there). 

Regarding the nested `context`s, you're right in that they're not needed here. I find that by breaking up complex setups (like this one, with staking, registering, deposits, etc.) Solidity contract tests are easier to write and maintain, since different features will usually be tested at different stages (e.g. attempting to `relayCall` before register, after removal or penalization, etc.).